src/main/java/Calculator.java

```diff
@@ -0,0 +1,79 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    Scanner scanner = new Scanner(System.in);
+    private final String ANSI_RED = "\u001B[31m";
+    private final String ANSI_RESET = "\u001B[0m";
+    private double tempArg;
+
+    public void numbersValidator() {
+        double input;
+        boolean inputIsValid = false;
+
+        do {
+            try {
+                input = scanner.nextDouble();
+                tempArg = input;
+                inputIsValid = true;
+            } catch (InputMismatchException e) {
+                System.out.print(ANSI_RED + "Wrong input! Try Again: " + ANSI_RESET);
```

Comment:
It's better to throw an exception in catch block, but in case of such a simple program it's OK to use sout